### PR TITLE
Position paper post

### PR DESCRIPTION
**Pull request by c-martinez**
Preview here: https://c-martinez.github.io/nl-rse.github.io/

### CODE REVIEW

File: _posts/2019-09-17-RasingResearchSoftware.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+image: '/img/Black_hole_-_Messier_87_crop_max_res.jpg'
+title: "Raising the Profile of Research Software: how this community action started"
```
Comment (nielsdrost) [inline _posts/2019-09-17-RasingResearchSoftware.md]: Perhaps add "Request For Comment:" or something to show we are requesting feedback.

**Comment by benvanwerkhoven** [comment]
At the top the blogpost reads "This blog post is work in progress", is that still the case?

**Comment by c-martinez** [comment]
Yes -- Yasemin set up a shared document for the blog. Once we have feedback, from the core group, and we cross post it on delft blog (and hopefully SSI blog) we'll remove the notice.

**Comment by benvanwerkhoven** [comment]
OK great!

**Pull request merged**